RadixWallet/Clients/ROLAClient/ROLAClient+Interface.swift

```diff
@@ -59,12 +59,12 @@ public struct AuthenticationDataToSignForChallengeResponse: Sendable, Hashable {
 
 // MARK: - ManifestForAuthKeyCreationRequest
 public struct ManifestForAuthKeyCreationRequest: Sendable, Hashable {
-	public let entity: EntityPotentiallyVirtual
-	public let newPublicKey: SLIP10.PublicKey
+	public let entity: AccountOrPersona
+	public let newPublicKey: Sargon.PublicKey
```

Comment:
Unfortunatly the terrible Gateway generated models have stolen `PublicKey` identifier, since those models are declared in the same module as iOS wallet, the Swift compiler always assume we mean `GatewayAPI.PublicKey` when we write `PublicKey`, so we MUST spell out `Sargon.PublicKey`. 

Not worth addressing now since Gateway will move into Sargon.... and this will become a non issue.